docs/howto/kubernetes-large-clusters.md

```diff
@@ -1,15 +1,18 @@
-# Large Kubernetes Clusters with aks-engine
+# Large Kubernetes Clusters with AKS-Engine
 
 ## Background
-Starting from aks-engine v0.3.0, aks-engine supports using exponential cloud backoff that is a feature of Kubernetes v1.6.6 and newer. Cloud backoff allows Kubernetes nodes to backoff on HTTP 429 errors that are usually caused by exceeding Azure API limits.
+
+Starting from AKS-Engine v0.3.0, AKS-Engine supports using exponential cloud backoff that is a feature of Kubernetes v1.6.6 and newer. Cloud backoff allows Kubernetes nodes to backoff on HTTP 429 errors that are usually caused by exceeding Azure API limits.
 
 ## To Use
+
 Declare your kubernetes cluster API model config as you normally would, with the following requirements:
-- You must be using at minimum the `v1.6.6` version of Kubernetes to have access to the `kubernetesConfig` configuration vectors exemplified in [examples/largeclusters/kubernetes.json](https://github.com/Azure/aks-engine/blob/master/examples/largeclusters/kubernetes.json). As long as you are using a version of aks-engine `v0.3.0` or newer, your kubernetes cluster specification will fulfill this minimum version requirement.
+- You must be using at minimum the `v1.6.6` version of Kubernetes to have access to the `kubernetesConfig` configuration vectors exemplified in [examples/largeclusters/kubernetes.json](https://github.com/Azure/aks-engine/blob/master/examples/largeclusters/kubernetes.json). As long as you are using a version of AKS-Engine `v0.3.0` or newer, your kubernetes cluster specification will fulfill this minimum version requirement.
```

Comment:
not sure how we need to address. This paragraph clearly references acs-engine v0.3.0 @jackfrancis thoughts?

Reply:
Let's fix this in a follow-up. The initial first pass was to just move stuff around to make it a little more manageable. In the second pass I intend to take a closer look at the documentation and remove any old docs (like the docs on SSH removed in this PR)